metadata-ingestion/src/datahub/ingestion/source/redshift/query.py

```diff
@@ -13,11 +13,13 @@ class RedshiftQuery:
     list_schemas: str = """SELECT database_name,
         schema_name,
         schema_type,
-        usename as schema_owner_name,
+        -- setting user_name to null as we don't use it now now and it breaks backward compatibility due to additional permission need
+        -- usename as schema_owner_name,
+        null as schema_owner_name,
         schema_option,
         NULL::varchar(255) as external_database
         FROM SVV_REDSHIFT_SCHEMAS as s
-        inner join pg_catalog.pg_user_info as i on i.usesysid = s.schema_owner
+        -- inner join pg_catalog.pg_user_info as i on i.usesysid = s.schema_owner
```

Comment:
Can we just delete this line and other commented out lines? Also, how come we need `schema_owner_name` at all? Could we just remove it?